components/web/services/org.wso2.carbon.apimgt.authenticator/src/main/java/org/wso2/carbon/apimgt/authenticator/AuthenticatorAPI.java

```diff
@@ -87,19 +87,25 @@ public Response authenticate(
 
             String part1 = accessToken.substring(0, accessToken.length() / 2);
             String part2 = accessToken.substring(accessToken.length() / 2);
-            NewCookie cookie = AuthUtil.cookieBuilder(AuthenticatorConstants.TOKEN_1, part1, appContext, true, false);
-            NewCookie cookie2 = AuthUtil
-                    .cookieBuilder(AuthenticatorConstants.TOKEN_2, part2, restAPIContext, true, true);
+            NewCookie appContextcookie1 = AuthUtil
+                    .cookieBuilder(AuthenticatorConstants.TOKEN_1, part1, appContext, true, false, "");
+            NewCookie appContextcookie2 = AuthUtil
```

Comment:
Shall we use descriptive variable names .  appContextcookie1 appContextcookie2  doesn't make much sense.